examples/jsf/numberguess/README.md

```diff
@@ -45,7 +45,7 @@ Make sure you have assigned the absolute path of your installation to the
 
 3. Now you can view the application at <http://localhost:8080/weld-numberguess>.
 
-Deploying to standalone Tomcat 8
+Deploying to standalone Tomcat 9
```

Comment:
Can the examples be Tomcat version-agnostic the same way as WildFly version-agnostic?

Reply:
Probably could but depending on Java version you want certain version of Tomcat.
The example should be definite in with which version it will run correctly. And 9 *seems* to do the trick (working on both, Java 8 and 9+).

Reply:
OK, thanks!